submodules/command_ingest/__init__.py

```diff
@@ -1,114 +1,102 @@
-                    else:
-                        args.append(argtype(cmd[i + 1]))
+                    print("REACHED")
+                    print(associated(*arguments))  # Actually run the command
                 except:
-                    # There was an error when converting an argument
-                    logger.debug("Unable to parse argument for command.")
-                    total_errors += 1
-                    return
-            try:
-                # Actually execute the command
-                resp = registered_commands[cmd[0]]["function"](*args)
-                if resp is not None:
-                    # TODO: Enqueue as an event message
-                    pass
-            except Exception as e:
-                logger.error("Exception when executing command %s: %s" %
-                             (cmd[0], str(e)))
-                total_errors += 1
-                return
-            total_success += 1
-    else:
-        # The message was not intended for us.
-        # logger.debug('Invalid message')
-        pass
-
-# def start():
-#     global modules, m_aprs, m_gps, m_adcs, m_eps
-#     # modules = {'A':core,'B':m_aprs,'C':'iridium','D':'housekeeping','E':'log','F':'GPS'}
-#     m_aprs = {'a': aprs.enqueue, 'b': aprs.enter_normal_mode, 'c': aprs.enter_low_power_mode,
-#               'd': aprs.enter_emergency_mode}
-#     m_gps = {'a': gps.sendgpsthruaprs, 'b': gps.queryfield, 'c': gps.querygps, 'd': gps.querypastgps,
-#              'e': gps.getsinglegps, 'f': gps.enter_normal_mode, 'g': gps.enter_low_power_mode,
-#              'h': gps.enter_emergency_mode}
-#     # m_adcs = {'a': adcs.enter_normal_mode, 'b': adcs.enter_low_power_mode, 'c': adcs.enter_emergency_mode}
-#     # m_eps = {'a': eps.pin_on, 'b': eps.pin_off, 'c': eps.get_pdm_status, 'd': eps.get_board_status, 'e': eps.set_system_watchdog_timeout, 'f': eps.get_bcr1_volts, 'g': eps.get_bcr1_amps_a, 'h': eps.get_bcr1_amps_b, 'i': eps.get_board_telem, 'j': eps.enter_normal_mode, 'k': eps.enter_low_power_mode, 'l': eps.enter_emergency_mode}
-#     # m_imu = {'a': imu.enter_normal_mode, 'b': imu.enter_low_power_mode, 'c': imu.enter_emergency_mode}
-#     # m_iridium = {}  # BLANK - nothing useful is in iridium.py
-#     # m_housekeeping = {'a': housekeeping.enter_normal_mode, 'b': housekeeping.enter_low_power_mode, 'c': housekeeping.enter_emergency_mode}
-#     # m_telemetry = {'a': telemetry.gps_subpacket, 'b': telemetry.adcs_subpacket, 'c': telemetry.comms_subpacket, 'd': telemetry.system_subpacket}
-#     modules = {'B': m_aprs, 'F': m_gps}
-#     # modules = {'B': m_aprs, 'C': m_iridium, 'D': m_housekeeping, 'F': m_gps, 'G': m_eps, 'H': m_adcs, 'I': m_imu, 'J': m_telemetry}
-#     # core = {}
+                    # TODO: send error immediately
+                    print("ERR: bad function")
+        else:
+            # TODO: log that checksum is incorrect
+            print("ERR: incorrect checksum")
+
+
+"""
```

Comment:
replace this old start with a new `start()`. Actually configure and do things here. 
i.e.
- Initialize all globals here
- Start threads, etc.